data/mods/gen7/random-teams.ts

```diff
@@ -985,7 +985,8 @@ export class RandomGen7Teams extends RandomTeams {
 			// If Wishiwashi, use the school-forme's much higher stats
 			if (species.baseSpecies === 'Wishiwashi') baseStats = this.dex.getSpecies('wishiwashischool').baseStats;
 
-			let bst = baseStats.hp + baseStats.atk + baseStats.def + baseStats.spa + baseStats.spd + baseStats.spe;
+			let bst = species.bst;
+			if (species.baseSpecies === 'Wishiwashi') bst = this.dex.getSpecies('wishiwashischool').bst;
```

Comment:
Do we need both Wishiwashi checks?

Reply:
id say up to @TheImmortal if he wants them consolidated

Reply:
Yeah we don't need the first one anymore.